docs/tutorial/quantum_annealing.ipynb

```diff
@@ -0,0 +1,446 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a329bc68-f20a-43df-aa39-d6c624faa307",
+   "metadata": {},
+   "source": [
+    "Same procedure for $H_1$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5bb8afe1-4cc8-47bd-845a-e84212bf37b5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "H1 = qm_o.Hamiltonian()\n",
+    "for q1 in range(V):\n",
+    "    H1 -= qm_o.X(q1)\n",
+    "qt_H1= qt_transpiler.transpile_hamiltonian(H1)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aad9d0d9-f406-4f3d-9bdd-d9c264853746",
+   "metadata": {},
+   "source": [
+    "We can calculate the energy spectrum by using `eigenenergies` form QuTiP. We will examine how the eigenenergy of the Hamiltonian changes as the parameter $s$ varies from 0 to 1.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e268dbb2-0a52-4f9c-b310-b75a638f0a94",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qutip import *\n",
+    "slist = np.linspace(0.0, 1.0, 101)\n",
+    "Elists = [[] for i in range(2**V)]\n",
+    "\n",
+    "for s in slist:\n",
+    "    qt_H = s*qt_H0 + (1-s)*qt_H1\n",
+    "    evals = qt_H.eigenenergies()\n",
+    "    for i in range(2**V):\n",
+    "        Elists[i].append(evals[i])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dd3e97a1-caf7-4cb0-be77-3cb34117d9b4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for i in range(2**V):\n",
+    "    plt.plot(slist, Elists[i], label=f\"{i+1}th\")\n",
+    "\n",
+    "plt.xlabel(\"t\")\n",
+    "plt.ylabel(\"Eigenenergies\")\n",
+    "#plt.legend()\n",
+    "plt.show()\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "42a44777-e227-43de-9e88-61ea5597e02e",
+   "metadata": {},
+   "source": [
+    "We can verify if the state corresponding to the ground energy is the optimal solution for the max-cut problem. According to the plot, there are four degenerate solutions. Let's examine the first eigenstate"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "55b79d48-f765-4c13-8a97-761a66eb5bd5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "evals, eigstates = qt_H0.eigenstates()\n",
+    "eket0 = eigstates[0]\n",
+    "state_vector = eket0.full().flatten()\n",
+    "indices_with_one = np.where(np.abs(state_vector) == 1.0)[0]\n",
+    "\n",
+    "# Convert indices to their binary representation\n",
+    "qubit_states = [\n",
+    "    np.array(list(map(int, format(index, f'0{V}b'))))\n",
+    "    for index in indices_with_one\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "54eced75-b0c4-4bf0-ab3d-32d0d0dd2c18",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Transform the qutip's state back to sampleset format\n",
+    "from qamomile.core.bitssample import BitsSample, BitsSampleSet\n",
+    "sample = BitsSample(1,qubit_states[0])\n",
```

Comment:
I think checking the grand state of final Hamiltonian is too trivial, so I think it is better to plot final state after annealing